extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/dto/totp/VaultTOTPCreateKeyBody.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.vault.runtime.client.dto.totp;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.quarkus.vault.runtime.client.dto.VaultModel;
+
+public class VaultTOTPCreateKeyBody implements VaultModel {
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
It is funny I have never spotted that I was sending `null` to vault in the other use cases. 
I would suggest making it a global default for the extension with `mapper.setSerializationInclusion(Include.NON_NULL);` rather than having to specify it on each field.